relion/protocols/protocol_subtract.py

```diff
@@ -171,14 +311,20 @@ def createOutputStep(self):
 
         self.reader = convert.createReader(alignType=ALIGN_PROJ)
         tableName = 'particles@' if self.IS_GT30() else ''
+        if self.isRelionInput:
+            doClone = False
+        else:
+            doClone = True
+            self.reader.setPixelSize(3.5)
```

Comment:
Seriously, what the heck is this pixel size?

Reply:
Sorry, this line is a leftover, It is not needed and therefore I delete it.